playbooks/tasks/portal-role-task.yml

```diff
@@ -28,10 +28,12 @@
   # Do not run this in parallel (use throttle = 1) so that there are not
   # created and used multiple JWKS data for a single cluster
 - block:
-    - name: Load JWKS data fresh from LastPass
-      include_tasks: tasks/lastpass-load-json-note.yml
+    - name: Include loading JWKS data fresh from secrets storage
+      include_tasks: "tasks/secrets-storage-common-load-yaml-or-json.yml"
```

Comment:
same, what are the quotations for? 

since this is different than the standard use case we should have a comment explaining the unique need.

Reply:
Removed quotes.
It was leftover from refactorings.